mlflow/metrics/metric_definitions.py

```diff
@@ -57,7 +55,6 @@ def _toxicity_eval_fn(predictions, targets, metrics):
     if not _validate_text_data(predictions, "toxicity", "predictions"):
         return
     try:
-        _logger.info("Loading toxicity metric:")
         import evaluate
 
         toxicity = evaluate.load("toxicity", module_type="measurement")
```

Comment:
can we take advantage of the `cache_dir` here so that it doesn't re-download after checking? or maybe some of the other configs of `evaluate.load()`

https://huggingface.co/docs/evaluate/v0.1.2/en/package_reference/loading_methods#evaluate.load.cache_dir

Reply:
i think `cache_dir` here is already used by default, the parameter is just to specify a different path

Reply:
also seems this is just for storing "temporary predictions and references"

Reply:
Cached with haru's PR: https://github.com/mlflow/mlflow/pull/10007